lib/makeTwemojiRenderer.js

```diff
@@ -60,4 +53,29 @@ function merge() {
 		}
    }
    return out
+}
+
+function getSource(options, icon){
```

Comment:
We should fold this function back into renderTwemoji, since the only change that needs to happen is on line 35

Reply:
https://github.com/appfigures/react-easy-emoji/pull/26/commits/167ab40e66d47faf4c5e1005c9488bc78ae765f6